qa/workunits/fs/cephfs_mirror_ha_gen.sh

```diff
@@ -25,7 +25,8 @@ trap cleanup EXIT
 configure_peer()
 {
     ceph mgr module enable mirroring
-    ceph fs snapshot mirror enable $PRIMARY_FS
+    # Ensure mirroring module is loaded
+    while ! ceph fs snapshot mirror enable $PRIMARY_FS; do sleep 5; done
```

Comment:
I'd use some kind of listing or status reporting command for this -- it seems like `ceph fs snapshot mirror daemon status` might be a good candidate -- to make the pure workaround nature of the retry loop clear and proceed with the actual test logic after.

I understand that you have already tested this though so feel free to ignore this nit.